leptonai/cli/pod.py

```diff
@@ -203,14 +211,22 @@ def list_command(pattern):
     table.add_column("status")
     table.add_column("ssh command")
     table.add_column("TCP port mapping")
+    table.add_column(
+        "TCP port mapping \n (defaults to the port that Jupyterlab listens on)",
+        justify="center",
+    )
     table.add_column("created at")
-    for pod, ssh_port, tcp_port, pod_ip in zip(pods, ssh_ports, tcp_ports, pod_ips):
+    for pod, ssh_port, tcp_port, tcp_port_jupyterlab, pod_ip in zip(
+        pods, ssh_ports, tcp_ports, tcp_ports_jupyterlab, pod_ips
+    ):
+        print(1)
         table.add_row(
             pod.metadata.name,
             pod.spec.resource_requirement.resource_shape,
             pod.status.state,
             f"ssh -p {ssh_port[1]} root@{pod_ip}" if pod_ip is not None else "N/A",
             f"{tcp_port[0]} -> {tcp_port[1]} \n(pod  -> client)",
+            f"{tcp_port_jupyterlab[0]} -> {tcp_port_jupyterlab[1]} \n(pod  -> client)",
```

Comment:
it appears if a pod has tcp, ssh, but not jupyter, the table will be messed up since the length of them are not the same and we can't use index to map them

Reply:
we may have historical pods like this